frontend/src/ui/components/StepBar/StepBar.tsx

```diff
@@ -47,10 +47,10 @@ export const StepBar: React.FC<StepBarProps> = ({
 }: StepBarProps) => (
   <div className="step-bar">
     {[...Array(count).keys()].map(num => (
-      <>
+      <React.Fragment key={`registration_stepbar_${num}`}>
```

Comment:
Thanks! This console error was pretty annoying, but I assumed you'll be addressing this anyway after the rebase :)